src/lib.rs

```diff
@@ -103,22 +102,22 @@ impl SafetyContext {
         for syscall in base_syscalls {
             if !rules.contains_key(&syscall) {
                 let rule = Rule::new(syscall);
-                rules.entry(syscall)
-                    .or_insert_with(Vec::new)
-                    .push(rule);
+                rules.entry(syscall).or_insert_with(Vec::new).push(rule);
```

Comment:
I have my editor set to auto-run `rustfmt` on save, so this change came about because of that. I did put the formatting update as its own first commit so it should be easy to remove if you prefer it be skipped.

Reply:
yes, unfortunately rustfmt makes a lot of decisions that make these kinds of statements uglier (while at the same time putting .unwrap() on newlines) so I end up just running it on diffs and manually taking the suggestions, which is not optimal.